### ISSUE TITLE
Android Auto sensors display

### ISSUE THREAD

**Issue by woniol** [home-assistant/android#3248]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

I'm after my first tests of Android Auto support. I looks really promising.
Would it be possible to add there sensors display in addition to all the currently available domains?
Posiibility of favourites selection would be needed for that as well, because of the possible number of sensor entieties.

**Describe alternatives you've considered, if any**

**Additional context**

**Comment by dshokouhi** [comment]
> Posiibility of favourites selection would be needed for that as well, because of the possible number of sensor entieties.

already requested https://github.com/home-assistant/android/issues/3219

**Comment by dshokouhi** [comment]
so asides from `sensor` and `binary_sensor` is there any other non actionable domains? `camera` won't qualify as we cannot show an image like that in the car.

**Comment by dshokouhi** [comment]
There is one issue here, we can only display a certain amount of entities. The grid item list also only allows certain character lengths, its also not recommended to display items that have no action which a sensor has no action. I am not sure this is something we should add given the limitations and considerations but lets see if other users really need this.

**Comment by GSzabados** [comment]
Then I have no idea how the Weather & Radar app has permission to display weather radar data on a map...

Anyhow, sensor data would be great. Groups would be good also. I have some grouped lights, which would be easier to control.

**Comment by dshokouhi** [comment]
> Then I have no idea how the Weather & Radar app has permission to display weather radar data on a map...


Displaying a map is actually something a POI or Navigation app can use, most likely they are a POI app whereas we are a IOT app

> Anyhow, sensor data would be great

I am hesitant to add a `Sensor` category because for most user they will never see all of them given that each vehicle has its own list limit which will impact this. I can see if it works for a sensor/binary sensor under favorites.

> Groups would be good also. I have some grouped lights, which would be easier to control.

please open a separate feature request for this as this request is about adding support for sensors. In the meantime its nothing a template entity can't fix :)


> Other groups, like motion sensors, I believe they fall under the binary_sensor category.

thats not a group ;)

https://www.home-assistant.io/integrations/group/

**Comment by GSzabados** [comment]
> I am hesitant to add a `Sensor` category because for most user they will never see all of them given that each vehicle has its own list limit which will impact this. I can see if it works for a sensor/binary sensor under favorites.

I agree with you on this. The amount of Sensors would be too many, only a few in the Favourites would work well.

**Comment by dshokouhi** [comment]
@GSzabados a bit off topic but old style groups do not work, try to recreate your light groups using the UI. Then the lights will be part of the `light` domain and you can add them without needing to update the app. A bit better this way so you can also dim them as a whole etc...
